airbyte-integrations/bases/base-typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TypeAndDedupeOperationValve.java

```diff
@@ -15,27 +15,21 @@
  */
 public class TypeAndDedupeOperationValve extends ConcurrentHashMap<AirbyteStreamNameNamespacePair, Long> {
 
-  private static final long TWO_MINUTES_MILLIS = 1000 * 60 * 2;
-
-  private static final long FIVE_MINUTES_MILLIS = 1000 * 60 * 5;
-
-  private static final long TEN_MINUTES_MILLIS = 1000 * 60 * 10;
-
-  // 15 minutes is the maximum amount of time allowed between checkpoints as defined by
-  // The Airbyte Protocol
+  private static final long ZERO_MINUTES_MILLIS = 0;
```

Comment:
I know I initially said zero, but (sorry!) after thinking about it, I think there is a pretty good likelihood that the first and second `if` conditions occur in less than 1 millisecond, so I think making this negative should account for that
```suggestion
  private static final long NEGATIVE_MILLIS = -1;
```